src/app/main/main.component.ts

```diff
@@ -137,6 +146,12 @@ export class MainComponent implements OnInit, OnDestroy {
     this.updateStyles();
   }
 
+  ngAfterViewInit() {
+    setTimeout( () => {
+        this.dataFormGroup.patchValue({ sql: "SELECT * FROM `" + this.project + "." + this.dataset + "." + this.table + "`"});
+    },0)
```

Comment:
I'm still planning to have a look at why this timeout is necessary, but I haven't had time yet and I wouldn't be opposed to merging in in the meantime.

However, I would have expected patching the form group in a timeout, outside an angular zone, to be error-prone by itself. Maybe something like this?

```js
setTimeout(() => {
  this._ngZone.run(() => {
    this.dataFormGroup.patchValue({ ... });
  });
}, 0);
```

Reply:
Done.